### ISSUE TITLE
qtmir-git doesn't build.. cannot find -lpthreads?

### ISSUE THREAD

**Issue by Danct12** [vanyasem/Lomiri-Arch#21]
Running cmake gives me this:
https://paste.ubuntu.com/p/b2sSWwfDj2/

However, CMakeError.log is a little bit different..
https://paste.ubuntu.com/p/skNBpFr24T/

Seems like it is giving me `/usr/bin/ld: cannot find -lpthreads`, not sure why. Any help would be appreciated.

**Comment by vanyasem** [comment]
I don't think that should be an issue, there were no commits to qtmir since I did a rebuild last time

**Comment by Danct12** [comment]
Hmm, weird then, I was building in a new chroot.. :\

**Comment by vanyasem** [comment]
I will check the package some time this week

**Comment by vanyasem** [comment]
could you please not use the ubports name and logo in your organization as it clearly has nothing to do with ubports (https://github.com/ubports-arch). Or at least make it clear that it's not official (see https://t.me/UBports_Lomiri/10617)

**Comment by vanyasem** [comment]
also you could've just made a PR to this repo, but well i guess it's up to you

**Comment by Danct12** [comment]
I did not make that group, one of the other guy did.

**Comment by Danct12** [comment]
Is the group alright now? Also, I haven't made anything worth a PR yet (removing check to get around QEMU's segfault).
